account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js

```diff
@@ -0,0 +1,32 @@
+//-*- coding: utf-8 -*-
+//© 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.account_multiple_account_type_reconciliation = function(instance) {
+	instance.web.account.bankStatementReconciliationLine.include({
+
+		selectMoveLine : function(mv_line) {
```

Comment:
I am missing any comments on what this function is doing, or how it implements the requested functionality.